src/main/java/com/toptrumps/core/DeckParser.java

```diff
@@ -1,4 +1,67 @@
 package com.toptrumps.core;
 
+
+import java.io.*;
+import java.util.ArrayList;
+
+
+// TODO: Remove this class
 public class DeckParser {
+
+    private ArrayList<Card> cards;
+
+    public DeckParser() {
+        buildDeck(); // calls the build deck method
+    }
+
+    private void buildDeck() {
+        cards = new ArrayList<>(); // creates a new array list
+
+        // creates a new File called cardsFile at the location of the file to be read in.
+        File cardsFile = new File("src/main/resources/assets/StarCitizenDeck.txt");
+
+        try {
+            FileReader fr = new FileReader(cardsFile);
+            BufferedReader br = new BufferedReader(fr);
```

Comment:
Yeeeees, great use of the BufferedReader! Increases performance while doing I/O.